pkg/ingester/shipper_test.go

```diff
@@ -305,27 +305,27 @@ func TestReadMetaFile(t *testing.T) {
 		// Create TSDB directory without meta file
 		dpath := t.TempDir()
 
-		_, err := readShipperMetaFile(dpath)
-		fpath := filepath.Join(dpath, shipperMetaFilename)
+		_, err := readThanosShipperMetaFile(dpath)
```

Comment:
We should rename this test to `TestReadThanosMetaFile` and replicate the same test for the new Mimir's meta.